pkgs/development/python-modules/py-air-control/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonPackage, coapthon3, fetchPypi, lib, pycryptodomex }:
```

Comment:
Is this useful as a Python module, or is it an application (i.e. is it really just for the "airctrl" CLI)?

If the former, this is great :)

If the latter, consider putting this elsewhere in the tree and then using "python3Packages.callPackage" from "all-packages.nix" instead.

Reply:
It's the former.

To give more context: I am using `py-air-control` as a library in [another Python module](https://github.com/urbas/py-air-control-exporter). Here's [an example of how it's used](https://github.com/urbas/py-air-control-exporter/blob/master/py_air_control_exporter/metrics.py#L5).

The final goal is to continuously measure air quality, publish the measurements to Prometheus, and visualize in Grafana (all in NixOS on a Raspberry Pi). I hope this makes sense.

Reply:
Sure! Nice :)

(ooh, I really want to start doing air quality measurement...)